src/Avalonia.Controls/TextBox.cs

```diff
@@ -283,7 +283,14 @@ public string SelectedText
                 }
 
                 _undoRedoHelper.Snapshot();
-                HandleTextInput(value);
+                if (value != "")
```

Comment:
What happens in case of `myFavoriteTextBox.SelectedText = null` ?

Reply:
Right before this block of code, we check if `value == null` and `return` if it is.

I just checked WPF: Setting `SelectedText = null` causes a crash, so WPF does not support setting `SelectedText` to `null` at all -- Avalonia at least does not crash.

If ya'll want `.SelectedText = null` to clear the text instead of doing nothing, I can change this to `string.IsNullOrEmpty(value)`. Easy enough to adjust.

Reply:
I think `SelectedText = null` should probably either throw an `ArgumentException` or do the same as an empty string. Just doing nothing seems like it could cause confusion.

Reply:
:+1: I made `SelectedText = null` do the same as an empty string and added a unit test for that case, too.